seed/challenges/01-responsive-web-design/css-flexbox.json

```diff
@@ -242,9 +247,10 @@
       "title": "Using the Flex-direction Property to Make a Row",
       "description": [
         "Adding <code>display: flex</code> to an element turns it into a flex container. This makes it possible to align any children of that element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to row or column. Creating a row will align the children horizontally, and creating a column will align the children vertically.",
+        "Other options for flex-direction are row-reverse and column-reverse",
         "<strong>Note</strong><br>The default value for the <code>flex-direction</code> property is row.",
```

Comment:
Two minor edits for this sentence - put `<code>` tags around flex-direction, and add a period to the end of the sentence.